apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -125,6 +126,11 @@ function WelcomeTourFrame() {
 		} );
 	};
 
+	useEffect( () => {
+		// focus the Next/Begin button as the first interactive element when tour loads
+		setTimeout( () => tourStartFocusedRef.current?.focus() );
```

Comment:
`setTimeout` is a bit of necessary evil. We need the focusing to happen after `KeyboardNavigation` loads, which reacts only on a focused tour. I like the scope of that, not sure if we should move this initial focus event to `KeyboardNavigation` instead. 🤔

@razvanpapadopol